misaligned/ColorPairTests.cpp

```diff
@@ -0,0 +1,54 @@
+#include "ColorPairTests.h"
+#include "ColorPairFormatter.h"
+#include "ColorPairs.h"
+#include <cassert>
+#include <vector>
+#include <sstream>
+#include <string>
+
+void ColorPairTests::testMinorColorRepeatedInConsecutiveLines() {
+    std::istringstream stream(ColorPairFormatter::formatColorMap());
+    std::string line, prev = "";
+    while (std::getline(stream, line)) {
+        const char delimiter = '|';
+        size_t pos = line.rfind(delimiter);
```

Comment:
Parsing the delimiter position is a great way to check alignment as well
(if the delimiter is in the same position in every row)